src/lsp/superbol_free_lib/arg_utils.ml

```diff
@@ -0,0 +1,27 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Ezcmd.V2
+open EZCMD.TYPES
+
+let en'dis'able_switch ~name ~default =
```

Comment:
Why not, if that gives a shorter name. Did you understand the purpose of this function at first sight? Otherwise, yes we'll need another name (maybe so it is called ``Arg_utils.switch `enable_disable``).

Reply:
I used `switch` with an additional argument for the phrasing in the documentation.